code/game/objects/structures/tables_racks.dm

```diff
@@ -536,6 +536,15 @@
 		return ..()
 
 /obj/structure/table/reinforced/attackby(obj/item/weapon/W as obj, mob/user as mob, params)
+	if(istype(W,/obj/item/weapon/stock_parts/scanning_module))
+		playsound(get_turf(src), 'sound/items/Deconstruct.ogg', 50, 1)
+		var/obj/machinery/optable/OPT = new /obj/machinery/optable(src.loc)
+		var/obj/item/weapon/stock_parts/scanning_module/SM = W
+		OPT.rating = SM.rating
+		user.drop_item(W)
+		qdel(W)
+		qdel(src)

```

Comment:
Also doesn't this mean you could use this to hop reinforced tables reeeeally easily